projects/discussion-ui/src/lib/components/tag-all-discussion/tag-all-discussion.component.ts

```diff
@@ -64,12 +62,13 @@ export class TagAllDiscussionComponent implements OnInit {
         this.setPagination()
       },
       (err: any) => {
+        console.log('Error in fetching single tag details', err)
```

Comment:
it is better if you use console.error() in all places where we are logging error messages